handlers/localeCode/handler.go

```diff
@@ -0,0 +1,41 @@
+package localeCode
```

Comment:
Should the directory name and the package name be `localecode/localecode_test`?

Reply:
maybe we can remove the specific handlers, as the only thing they do now is to call the main handler with a specific value from an enum?